crates/flashblocks-archiver/src/main.rs

```diff
@@ -0,0 +1,96 @@
+use clap::Parser;
+use flashblocks_archiver::{Config, FlashblocksArchiver};
+use tracing::{error, info};
+
+#[derive(Parser)]
+#[command(name = "flashblocks-archiver")]
+#[command(about = "Archives flashblock messages from multiple builders to PostgreSQL")]
+struct Cli {
+    #[arg(long, help = "Path to configuration file")]
+    config: Option<String>,
+
+    #[arg(long, help = "Database URL (overrides config file)")]
+    database_url: Option<String>,
+
+    #[arg(long, help = "Comma-separated list of builder WebSocket URLs")]
+    builder_urls: Option<String>,
```

Comment:
Personal preference (feel free to ignore), I would drop the config file and configure the automatically via clap with the env setting. Makes it easier to reason about when all the config is in one place (env vars), vs. multiple